pytestqt/qtbot.py

```diff
@@ -315,10 +317,19 @@ def waitSignals(self, signals=None, timeout=1000, raising=None):
            blocker.wait()
 
         :param list signals:
-            A list of :class:`Signal` objects to wait for. Set to ``None`` to
-            just use timeout.
+            A list of :class:`Signal`s objects to wait for. Set to ``None`` to just use
+            timeout.
         :param int timeout:
             How many milliseconds to wait before resuming control flow.
+        :param str force_order:
+            "none": no order is enforced
```

Comment:
Please format those as a list, like in [here](https://github.com/pytest-dev/pytest-qt/blob/master/pytestqt/qtbot.py#L109). The results are [worth it](http://pytest-qt.readthedocs.io/en/1.11.0/reference.html#pytestqt.qtbot.QtBot.mouseRelease) IMO.